lib/src/imgcodecs/imgcodecs.dart

```diff
@@ -73,6 +73,7 @@ Uint8List imencode(
   if (!success.value) {
     throw CvException(ErrorCode.StsError.code, msg: "imencode failed, check your params");
   }
+  calloc.free(cExt);
```

Comment:
@rainyl shouldn't the free happen before the success.value check? currently cExt wont be freed if `imencode` fails

Reply:
Oh, you are right, will be fixed soon